internal/network/manifests_generator.go

```diff
@@ -81,10 +81,19 @@ spec:
         overwrite: true
 `
 
-const snoDnsmasqConf = `
-address=/apps.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
-address=/api-int.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
-address=/api.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
+const snoDnsmasqConf = `#!/usr/bin/env bash
+
+CLUSTER_FULL_DOMAIN="{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}"
+if [ -f /etc/default/cluster-domain ]; then
```

Comment:
What is this file?

Reply:
This is the way to override domain as you can't verify what domain you have

Reply:
Then this file should probably follow a similar format to the other files in `/etc/default` (such as `/etc/default/nodeip-configuration`), e.g.:

```env
VAR_NAME=<value>
```

Reply:
Also let's name this file something more descriptive, like `openshift-sno-dns-cluster-name-domain-override`, and have it have 2 variables: 

CLUSTER_NAME_OVERRIDE
CLUSTER_BASE_DOMAIN_OVERRIDE

Reply:
would not go this way actually, the format is for setting envs, we don't need it so why to make it format that we don't actually will use

Reply:
/etc/default/nodeip-configuration was just added by networking team, the decision was made to make it convenient for them.

Reply:
>would not go this way actually, the format is for setting envs, we don't need it so why to make it format that we don't actually will use

This is the accepted format for files in `/etc/default`, regardless of OpenShift, I didn't make it up

Reply:
I mean there is no specified format for files in /etc/default, we can use another folder if it helps

Reply:
Also pretty sure that files in `/etc/default` get automatically sourced by systemd, so you don't even have to source it - you'll automatically get those vars in the service. But I'm not 100% sure about thatg